Refactor Inspector resource group resource to use keyvaluetags package (or not) (#11573)
terraform-providers/terraform-provider-aws: website/docs/r/inspector_resource_group.html.markdown

layout: "aws"
page_title: "AWS: aws_inspector_resource_group"
description: |-
  Provides an Amazon Inspector resource group resource.
---

# Resource: aws_inspector_resource_group

> Provides a Amazon Inspector resource group resource.

Reviewer: ```suggestion
Provides an Amazon Inspector resource group resource.
```

The following arguments are supported:

> * `tags` - (Required) The tags tags (key and value pairs) that are used to select the EC2 instances to be included in an [Amazon Inspector assessment target](/docs/providers/aws/r/inspector_assessment_target.html).

Reviewer: ```suggestion
* `tags` - (Required) Key-value map of tags that are used to select the EC2 instances to be included in an [Amazon Inspector assessment target](/docs/providers/aws/r/inspector_assessment_target.html).
```
